### PR TITLE
DEP Upgrade installer dependencies

### PR DESCRIPTION
**Pull request by sabina-talipova**
### Parent issue
- https://github.com/silverstripe/silverstripe-installer/issues/335

### CODE REVIEW

**Comment by sabina-talipova** [comment]
Required new version of `psr/simple-cache` , installed in [silverstripe-config PR](https://github.com/silverstripe/silverstripe-config/pull/71)

File: composer.json [hunk 1]
```diff
@@ -44,4 +44,4 @@
     },
     "minimum-stability": "dev",
     "prefer-stable": true
-}
+}
```
Comment (michalkleiner) [inline composer.json]: Despite not having the 'new line at the end of files' requirement in the [coding conventions](https://docs.silverstripe.org/en/4/contributing/coding_conventions/#coding-conventions), I think it's a good practice to keep the new line at the end of files. It also helps produce nicer git diffs in files where it matters.

Comment (dhensby) [reply composer.json]: We need this in the linting rules

**Review by GuySartorelli — APPROVED** [review]
Looks good. Tests run and pass locally with the config PR installed.

**Pull request merged**